hoomd/md/bond.py

```diff
@@ -381,3 +381,79 @@ def set_from_file(self, bondname, filename):  # noqa
                             rmin=rmin_table,
                             rmax=rmax_table,
                             coeff=dict(V=V_table, F=F_table, width=self.width))
+
+
+class Tether(Bond):
+    r"""Tethering bond potential.
+
+    :py:class:`Tether` specifies a Tethering potential energy between two
+    particles in each defined bond.
+
```

Comment:
Please include a citation to the paper that developed or first used this potential.